index.html

```diff
@@ -1802,10 +1804,10 @@ <h2 id="state-transitions" class="informative">
         </p>
         <section>
           <h2>
-            <dfn>updateWith()</dfn> method
+            <dfn data-lt="updateWith(details)">updateWith()</dfn> method
           </h2>
           <p>
-            The <a>updateWith</a> method MUST act as follows:
+            The <a>updateWith(details)</a> method MUST act as follows:
```

Comment:
Now it is inconsistent as the rest of the algorithm references `<var>d</var>`. This also doesn't address the `<var>`-in-updateWith vs. `<code>`-in-complete issue.